### PR TITLE
Git Basics: replaced screenshots with equivalent text and code snippets

### LINKED ISSUE #27904: Git Basics: <Accessibility of screenshots>
### Checks

- [X] This is not a duplicate of an existing issue (please have a look through our [open issues list](https://github.com/TheOdinProject/curriculum/issues) to make sure)
- [X] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
- [ ] Would you like to work on this issue?

### Describe your suggestion

I'm a low-vision user and this is the first page that I had some difficulty receiving the lesson information due to heavily relying on screenshots. In step 2 of Create the Repository, I am directed to click the button shown in the screenshot. I was able to enlarge the image quite a bit to get the information but a written description of the button itself and where it is located would be helpful. In the rest of the lesson I was unable to use the screenshots of the shell outputs to compare to my local output. It would be some work but it would greatly help if actual text was used instead of the screenshot. Most steps included key information of what to look for in my output which was helpful to confirm I was on the right track. My suggestion moving forward is to use text instead of screenshots if the screenshot is essentially text. Thanks!

### Path

Foundations

### Lesson Url

https://www.theodinproject.com/lessons/foundations-git-basics

### (Optional) Discord Name

_No response_

### (Optional) Additional Comments

_No response_

### PR DESCRIPTION
**Pull request by kol3x**
## Because

The lesson assignment used a bunch of screenshots, hurting accessibility.

## This PR

+ Deleted screenshot files
+ Added some text fixes and code snippets to compensate for screenshots.

## Issue

Closes #27904

## Additional Information

I was about to commit this PR and noticed that my PR is deleting main README file for some reason... I returned it but github still acts like I changed something in it :sweat_smile: 

## Pull Request Requirements

-   [x] I have thoroughly read and understand [The Odin Project curriculum contributing guide](https://github.com/TheOdinProject/curriculum/blob/main/CONTRIBUTING.md)
-   [x] The title of this PR follows the `location of change: brief description of change` format, e.g. `Intro to HTML and CSS lesson: Fix link text`
-   [x] The `Because` section summarizes the reason for this PR
-   [x] The `This PR` section has a bullet point list describing the changes in this PR
-   [x] If this PR addresses an open issue, it is linked in the `Issue` section
-   [x] If any lesson files are included in this PR, they have been previewed with the [Markdown preview tool](https://www.theodinproject.com/lessons/preview) to ensure it is formatted correctly
-   [x] If any lesson files are included in this PR, they follow the [Layout Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md)

### CODE REVIEW

**Comment by kol3x** [comment]
markdown playing with me -_-

File: git/foundations_git/git_basics.md
Comment (thatblindgeye) [inline git/foundations_git/git_basics.md:48]: Thinking maybe we tweak this instruction a little more:

```suggestion
1. <span id="origin-push"></span>That's it! You have successfully connected the repository you created on GitHub to your local machine. To test this, `cd` into the new **git_test** folder that was downloaded and then enter `git remote -v` on your command line. You should see an output similar to the following, where `USER-NAME` is your GitHub username:
  ```bash
  origin  git@github.com:USER-NAME:git_test.git (fetch)
  origin  git@github.com:USER-NAME:git_test.git (push)
  ```
  This displays the URL of the repository you created on GitHub, which is the remote for your local copy. <span id="default-remote"></span>You may have also noticed the word **origin** at the start of the `git remote -v` output, which is the name of your remote connection. The name "origin" is both the default and the convention for the remote repository. But it could have just as easily been named "party-parrot" or "dancing-banana". (Don't worry about the details of origin for now; it will come up again near the end of this tutorial.)
```

File: git/foundations_git/git_basics.md
Comment (thatblindgeye) [inline git/foundations_git/git_basics.md:53]: We can remove this bash block.

**Review by thatblindgeye — CHANGES_REQUESTED** [review]
A lot of change requests, but a lot of them are super quick ones that mainly relate to removing bash blocks that we don't really need. Since instruction items say inline "Enter this command `some command`.", the bash block isn't really necessary if it's just stating the same command again.

File: git/foundations_git/git_basics.md
Comment (kol3x) [reply git/foundations_git/git_basics.md:29]: Right, I thought about that.

File: git/foundations_git/git_basics.md
Comment (MaoShizhong) [inline git/foundations_git/git_basics.md:81]: We won't need the MD033 disable anymore because `<kbd>` is in the allowed elements array in our config.

We can go back to using backticks for the file names (which we should use instead of `*`) by just making the backtick in the `<kbd>` tag use the right HTML entity instead of the raw character. That will satisfy both the linter and Kramdown when we render the web page.

```suggestion
1. Go back to your terminal or if you're using Visual Studio Code you can open the built-in terminal by pressing <kbd>Ctrl</kbd> + <kbd>&grave;</kbd> (backtick). Then type `git status`. Notice how the output is similar to when we created our `hello_world.txt` file before adding it to the staging area, except the `README.md` file is listed in a section titled "Changes not staged for commit". The meaning is similar to the "Untracked files" section in that the file is not yet added to the staging area.
```

**Review by MaoShizhong — CHANGES_REQUESTED** [review]

File: git/foundations_git/git_basics.md
Comment (MaoShizhong) [reply git/foundations_git/git_basics.md:153]: Strange - Not getting any linting errors with this block unindented. Can't see any rules that would trigger it either, unless at some point the surroundings looked different.

**Comment by kol3x** [comment]
@MaoShizhong Done! It doesn't work in preview tool, but I suppose will work on prod?
image

File: git/foundations_git/git_basics.md
Comment (thatblindgeye) [inline git/foundations_git/git_basics.md:33]:
```suggestion
1. This will redirect you to your new repository on GitHub. You should see a layout that includes a tab list of different views (such as "Code", "Pull Requests", and so on) and your repository name along the top of the page, along with a lot of other content elsewhere on the page. To get ready to copy (clone) this repository onto your local machine, click the "Code" button which should be in the section **below** your repository name - note that you should not click the "Code" *tab* that appears above your repository name. Then select the SSH option in the "Clone" section, and copy the line below it. **NOTE: You MUST click the SSH option to get the correct URL.**
```
Lemme know what you think of the above addition. Realized it may be confusing that there's a "Code" tab and button on the same page.

**Comment by MaoShizhong** [comment]
> @MaoShizhong Done! It doesn't work in preview tool, but I suppose will work on prod? image

Oops, that's my mistake! I listed the wrong suggestion - instead of `&grave;`, it should be `&#96;` which does work in the markdown preview tool. For whatever reason, Kramdown accepts HTML codes but not entities, which are parsed as plaintext instead. Apologies!
Would you mind also remove the comment on line 79? I also missed that line in the previous suggestion selection.

File: git/foundations_git/git_basics.md
Comment (thatblindgeye) [inline git/foundations_git/git_basics.md:79]: This can be removed if we're not keeping the comment to disable the rule.

**Review by thatblindgeye — COMMENTED** [review]
This is looking great 💪🏼 Couple more comments below, and then let me know if you wanted to add some additional verbiage about the location of the "Create repository" instructions on smaller viewports.

File: git/foundations_git/git_basics.md
Comment (kol3x) [reply git/foundations_git/git_basics.md]: @MaoShizhong you are right, I reverted the indentation now

File: git/foundations_git/git_basics.md
Comment (kol3x) [reply git/foundations_git/git_basics.md:33]: I would delete this part "You should see a layout that includes a tab list of different views (such as "Code", "Pull Requests", and so on) and your repository name along the top of the page, along with a lot of other content elsewhere on the page." to not focus attention on that other Code button. 

And instead of "click the "Code" button which should be in the section **below** your repository name - note that you should not click the "Code" *tab* that appears above your repository name." I would say "Click the green "Code" button at the bottom right of the repository name."

**Comment by kol3x** [comment]
> This is looking great 💪🏼 Couple more comments below, and then let me know if you wanted to add some additional verbiage about the location of the "Create repository" instructions on smaller viewports.

i checked and it seems like the "+" button is always in the top-right corner somewhere, unless you are on mobile, which I think learner is not supposed to be on. So the the current instruction should be sufficient

File: git/foundations_git/git_basics.md
Comment (thatblindgeye) [reply git/foundations_git/git_basics.md:33]: So you're thinking something more along the lines of:

> 1. This will redirect you to your new repository on GitHub. To get ready to copy (clone) this repository onto your local machine, click the green "Code" button which should be below and to the right of your repository name. Then select the SSH option in the "Clone" section, and copy the line below it. **NOTE: You MUST click the SSH option to get the correct URL.**

That would probably be good. An alternative would be instead of mentioning its position based on the repo name, using the branch toggle or the repo files/folders, e.g. "click the green "Code" button, which should be to the right of the button displaying the current branch (typically it will display the `main` branch)" or "click the green "Code" button, which should be directly above the section that contains the files and folders in your directory"

I'll leave it up to you what makes the most sense from a user perspective, though.

**Comment by thatblindgeye** [comment]
>i checked and it seems like the "+" button is always in the top-right corner somewhere, unless you are on mobile, which I think learner is not supposed to be on. So the the current instruction should be sufficient

It'll not be displayed at certain viewport widths larger than mobile as well, such as if someone is splitting their monitor to have the browser open on one half and VS Code/something else open on the other. Adding instructions for this scenario isn't a blocker, though, especially since we don't currently have those instructions, so we can always revisit this if people do start voicing confusion.

File: git/foundations_git/git_basics.md
Comment (kol3x) [reply git/foundations_git/git_basics.md:33]: > So you're thinking something more along the lines of:
> 
> > 1. This will redirect you to your new repository on GitHub. To get ready to copy (clone) this repository onto your local machine, click the green "Code" button which should be below and to the right of your repository name. Then select the SSH option in the "Clone" section, and copy the line below it. **NOTE: You MUST click the SSH option to get the correct URL.**
> 
> That would probably be good. An alternative would be instead of mentioning its position based on the repo name, using the branch toggle or the repo files/folders, e.g. "click the green "Code" button, which should be to the right of the button displaying the current branch (typically it will display the `main` branch)" or "click the green "Code" button, which should be directly above the section that contains the files and folders in your directory"
> 
> I'll leave it up to you what makes the most sense from a user perspective, though.

I'm gonna go with relating the "Code" button position to branch name, as that doesn't change on smaller viewports

**Comment by kol3x** [comment]
> > i checked and it seems like the "+" button is always in the top-right corner somewhere, unless you are on mobile, which I think learner is not supposed to be on. So the the current instruction should be sufficient
> 
> It'll not be displayed at certain viewport widths larger than mobile as well, such as if someone is splitting their monitor to have the browser open on one half and VS Code/something else open on the other. Adding instructions for this scenario isn't a blocker, though, especially since we don't currently have those instructions, so we can always revisit this if people do start voicing confusion.

I improved directions for "New Repo" and "Code" buttons in the latest commits. 

Let me know if there is anything else to be improved.

**Review by thatblindgeye — APPROVED** [review]
🔥 This looks good. Before merging I just want to check whether this verbiage regarding cloning your repo is accurate, as I'm not sure if it matches what you had mentioned above about relating the position to the branch name:

> click the green "Code" button which should be below and to the right of your repository name

**Comment by kol3x** [comment]
> 🔥 This looks good. Before merging I just want to check whether this verbiage regarding cloning your repo is accurate, as I'm not sure if it matches what you had mentioned above about relating the position to the branch name:
> 
> > click the green "Code" button which should be below and to the right of your repository name

omg, I thought I went insane, but turns out I just didn't push this change, it was left hanging in my VSCode
Should be good now :)

**Review by thatblindgeye — APPROVED** [review]
It happens to the best of us! 😆 Thanks for making these updates 💪🏼

**Pull request merged**
